lib/server/utils.js

```diff
@@ -21,11 +21,13 @@ var utils = {
     },
     /**
      * @param filePath
+     * @param passphrase
      * @returns {{pfx}}
      */
-    getPFX: function (filePath) {
+    getPFX: function (filePath, passphrase) {
```

Comment:
I don't think so, and the PFX tests with no passphrase still seem to be passing.